### PR TITLE
Images branch

### PR DESCRIPTION
**Pull request by sabawaheed27**

### CODE REVIEW

File: aboutus.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About us</title>
+    <link rel="stylesheet" href="./styles/aboutus.css">
+    <link rel="stylesheet" href="./styles/global.css">
+</head>
+
```
Comment (Moshira89) [inline aboutus.html:11]: Looking good!

**Review by Moshira89 — APPROVED** [review]

**Review by Moshira89 — APPROVED** [review]

**Review by mvic89 — APPROVED** [review]
All good! only the header and footer that needs to be added.

**Comment by bushra-rauf** [comment]
i found on About Us title the u is smaller you have to capitaliaze it.

**Review by bushra-rauf — COMMENTED** [review]
line 7 u should be capitalize.

File: aboutus.html (hunk 1 above)
Comment (bushra-rauf) [reply aboutus.html:11]: yes its good

**Review by bushra-rauf — COMMENTED** [review]
in line 20 u is still u is in lowercase. you need to fix it

**Pull request merged**